### ISSUE TITLE
[Accessability] [Dark theme] Folder shared link size and modified columns text not visible

### ISSUE THREAD

**Issue by antonol68** [nextcloud/server#27371]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1.Go to Accessibility, enable dark theme
2.Share link from one folder
3.Open the shared link

### Expected behaviour
The size and modified columns text should be white 

### Actual behaviour
The size and modified columns text are grey and black

### Server configuration

Freebsd 12

Nginx 1.18.0

MariaDB 10.5.8

PHP 7.4.16

Nexcloud 21.0.2

Updated from an older Nextcloud

Where did you install Nextcloud from: Web

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
Browser:
Firefox 78.10.0 esr (64-bit)

Operating system:
Debian 10.9

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by MichaIng** [comment]
Confirmed.

The modification time is expected to be grey, same with file sizes < 20 MiB (on my instance, not sure if this is hardcoded). But file files > 20 MiB are shown white in the files app (right), but black on the share (left):
![share](https://user-images.githubusercontent.com/28480705/121205746-3671f200-c878-11eb-97d1-29036c323085.png)

**Comment by szaimen** [comment]
After a deeper investigation I found out that this check doesn't return true on public sites when you are not logged in
https://github.com/nextcloud/server/blob/09b38bce4ef12027cf4a899bb4213a4dca469351/apps/files/js/filelist.js#L1664

cc @nickvergessen for a fast fix on this one as I have no idea why it doesn#t work on public sites. Is maybe the Accessibility store or theme store not available?

BTW: this is a follow up of https://github.com/nextcloud/server/pull/21366

**Comment by szaimen** [comment]
> Is maybe the Accessibility store or theme store not available?

seems like this is the case

**Comment by szaimen** [comment]
Found a solution
